src/ReactWrapperComponent.jsx

```diff
@@ -12,63 +12,54 @@ import objectAssign from 'object.assign';
  * pass new props in.
  */
 export default function createWrapperComponent(node, options = {}) {
-  const spec = {
-
-    propTypes: {
-      Component: PropTypes.oneOfType([PropTypes.func, PropTypes.string]).isRequired,
-      props: PropTypes.object.isRequired,
-      context: PropTypes.object,
-    },
-
-    getDefaultProps() {
-      return {
-        context: null,
-      };
-    },
-
-    getInitialState() {
-      return {
+  class WrapperComponent extends React.Component {
```

Comment:
I think it would be nice if Enzyme users don't have to install `create-react-class` to use it.